plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -130,7 +130,11 @@ private ModelWrapper GenerateNonOverloadPage(PageViewModel page, FileModel model
             item.Metadata[SplitReferencePropertyName] = true;
             var newPage = ExtractPageViewModel(page, new List<ItemViewModel> { item });
             var newModel = GenerateNewFileModel(model, newPage, item.Uid);
-            var tree = ConvertToTreeItem(item);
+            var tree = ConvertToTreeItem(item,
+                new Dictionary<string, object>
+                {
+                    [Constants.PropertyName.Platform] = item.Platform,
+                });
```

Comment:
oops, no need to

---
In reply to: [102664712](https://github.com/dotnet/docfx/pull/1341#discussion_r102664712) [](ancestors = 102664712)